themes/custom/az_barrio/templates/content/node--az-person--full.html.twig

```diff
@@ -149,22 +149,28 @@
         {% if content.field_az_media_image.0 %}
           {{ content.field_az_media_image.0 }}
         {% endif %}
+          {% if content.field_az_address.0 %}
           <div class="card-body pb-0">
             {{ content.field_az_address.0 }}
           </div>
+          {% endif %}
+          {% if content.field_az_phones or content.field_az_email or content.field_az_links %}
           <ul class="list-group list-group-flush border-0 py-3">
           {% if content.field_az_phones %}
             {% for key, item in content.field_az_phones if key|first != '#' %}
               <li class="list-group-item border-0 py-0">{{ item }}</li>
             {% endfor %}
           {% endif %}
+          {% if content.field_az_email %}
```

Comment:
I think that this needs to be: `if content.field_az_email is defined`.
Phones and links are both arrays, so the current syntax which checks to see if they are empty works, but I don't think content.field_az_email is an array, so the if statement needs to check if it's defined. 

If this is correct, it would also apply to line 157.

Reply:
> I think that this needs to be: `if content.field_az_email is defined`.
> Phones and links are both arrays, so the current syntax which checks to see if they are empty works, but I don't think content.field_az_email is an array, so the if statement needs to check if it's defined.
> 
> If this is correct, it would also apply to line 157.

Could be.  I think @jwmoore1 is actively working on this issue but if you want to make a commit suggestion (or two) here we can try it.

Reply:
I was just looking at the docs here: https://twig.symfony.com/doc/3.x/tags/if.html

But I was not confident enough to submit it as a commit suggestion. It doesn't appear to be working on a local build.